### ISSUE TITLE
[feature request] Speed of sound / mach number

### ISSUE THREAD

**Issue by vinix38** [Qalculate/libqalculate#364]
I searched through the issues, but couldn't find it, so I am adding a new issue for it:
Would it be possible to add the speed of sound/mach number as a unit ?
I suggest that the app should use the value for a standard atmosphere, i.e., 340 m/s
(note aside: I really enjoy using qalculate, it is a really great tool !)

**Comment by hanna-kn** [comment]
I do not think it would be potentially confusing and misleading to add a fixed mach unit. It would be more appropriate to either add an constant for the approximate speed of sound in ordinary air at a specific temperature, or add a function for calculation of the speed of sound under specific conditions (e.g. a function which returns the approximate speed of sound in air in a specified temperature, and possibly also with a specified humidity and frequency, or perhaps a function which mimics flight instruments and makes assumptions based on pressure).

**Comment by lgommans** [comment]
I would also like to have a mach unit, simply at standard conditions, but I do see the concern.

I'm very new to qalc (and I only know the command line version), so I'll just have to ask: is there a way to set an option so that the user can indicate they just want standard conditions (pressure, temperature, air's gas mixture, etc. at sea level)? I'm imagining something like `set standard_conditions`.

Or could the unit be named appropriately to avoid any confusion, such as `mach_sealevel` ([sea-level standard](https://en.wikipedia.org/wiki/Standard_sea-level_conditions)) or `mach_STP` ([STP](https://en.wikipedia.org/w/index.php?title=Standard_temperature_and_pressure&oldid=1080382681))?

**Comment by vlad2305m** [comment]
For all ambiguous cases, you can add a custom unit yourself in the graphical app, and export the custom units file to share it with friends. I myself use it to define units for a video game, code is open source.
